components/Frame/Popover/NavNew.js

```diff
@@ -0,0 +1,225 @@
+  Button,
+  TeaserSectionTitle
+} from '@project-r/styleguide'
+import { HEADER_HEIGHT, HEADER_HEIGHT_MOBILE } from '../../constants'
+
+import withT from '../../../lib/withT'
+import withInNativeApp from '../../../lib/withInNativeApp'
+import { matchPath } from '../../../lib/routes'
+import SignIn from '../../Auth/SignIn'
+import { withMembership } from '../../Auth/checkRoles'
+import Footer from '../Footer'
+import SearchForm from '../../Search/Form'
+import Link from '../../Link/Href'
+import NavLink, { NavA } from './NavLink'
+import Sections from './Sections'
+
+const Nav = ({
+  me,
+  router,
+  expanded,
+  closeHandler,
+  children,
+  t,
+  inNativeApp,
+  inNativeIOSApp,
+  isMember,
+  onSearchSubmit
+}) => {
+  const active = matchPath(router.asPath)
+  const hasExpandedRef = useRef(expanded)
+  if (expanded) {
+    hasExpandedRef.current = true
+  }
+  return (
+    <>
+      <hr {...styles.hr} {...styles.hrFixed} />
+      <Center {...styles.container} id='nav'>
+        {hasExpandedRef.current && (
+          <>
+            {!me && (
+              <>
+                <div {...styles.signInBlock}>
+                  <SignIn style={{ padding: 0 }} />
+                </div>
+              </>
+            )}
+            {!isMember && (
+              <Button style={{ marginTop: 24 }} href='/pledge' black block>
+                {t('nav/becomemember')}
+              </Button>
+            )}
+            {me && (
+              <SearchForm
+                style={{ padding: 0 }}
+                reduced
+                onSearchSubmit={onSearchSubmit}
+              />
+            )}
+            <div {...styles.navSection}>
+              <div {...styles.navLinks}>
+                <NavLink
+                  route='index'
+                  active={active}
+                  closeHandler={closeHandler}
+                >
+                  {t('navbar/front')}
+                </NavLink>
+                <NavLink
+                  prefetch
+                  route='feed'
+                  active={active}
+                  closeHandler={closeHandler}
+                >
+                  {t('navbar/feed')}
+                </NavLink>
+                <NavLink
+                  route='discussion'
+                  active={active}
+                  closeHandler={closeHandler}
+                  hoverColor={colors.primary}
+                >
+                  {t('navbar/discussion')}
+                </NavLink>
+              </div>
+            </div>
+            <hr {...styles.hr} />
+            <div {...styles.navSection}>
+              <div style={{ color: colors.lightText }}>
+                <Link
+                  href='/sections'
+                  active={active}
+                  closeHandler={closeHandler}
+                  passHref
+                >
+                  <TeaserSectionTitle small>
+                    {t('nav/sections')}
+                  </TeaserSectionTitle>
+                </Link>
```

Comment:
The sections/Rubriken link is currently broken.

You're using the wrong link here. Remove `import Link from '../../Link/Href'` and import `Link` from `lib/routes`: `import { matchPath, Link } from '../../../lib/routes'`.

And try this:

```suggestion
                <Link
                  route='sections'
                  passHref
                >
                  <TeaserSectionTitle small onClick={closeHandler}>
                    {t('nav/sections')}
                  </TeaserSectionTitle>
                </Link>
```
(not a 100% sure that the onClick is triggered but if I remember our last test together correctly it was)

The path link you were using never supported active or close handler props. Maybe you intended to use the `NavLink`? But it probably can't handle the style correctly.

Reply:
TeaserSectionTitle accepts an onClick handle, which is how I used it in UserNav as well. so that should work